bpf/tc/attach.go

```diff
@@ -478,6 +502,12 @@ func RemoveQdisc(ifaceName string) error {
 	if err != nil {
 		return fmt.Errorf("failed to remove qdisc from interface '%s': %w", ifaceName, err)
 	}
+
+	// Remove the hash file of the program attached to the interface
+	if err = bpf.RemoveProgHash(ifaceName, "tc_"+hook); err != nil {
```

Comment:
Should also remove the file in `onInterfaceUpdate` if the interface is removed (as signaled by State=ifacemonitor.StateUnknown)